### PR TITLE
[CELEBORN-887] Option --config should take effect in celeborn-daemon.sh script

### PR DESCRIPTION
**Pull request by onebox-li**
### What changes were proposed in this pull request?
I find a little difficult to use `celeborn-daemon.sh` to get instance status, so I polish the usage and fix --config load.

### Why are the changes needed?
Ditto

### Does this PR introduce _any_ user-facing change?
Polish the `celeborn-daemon.sh` usage 

### How was this patch tested?
Manually test.

### CODE REVIEW

**Review by pan3793 — APPROVED** [review]
Thanks, this looks correct to me

File: sbin/celeborn-daemon.sh [hunk 1]
```diff
@@ -28,20 +28,22 @@
 #   CELEBORN_NO_DAEMONIZE   If set, will run the proposed command in the foreground. It will not output a PID file.
 ##
 
-usage="Usage: celeborn-daemon.sh [--config <conf-dir>] (start|stop|status) <celeborn-command> <celeborn-instance-number> <args...>"
+usage() {
+  echo "Usage: celeborn-daemon.sh [--config <conf-dir>] (start|stop|restart|status) <celeborn-class-name> <celeborn-instance-number> [args]"
+  echo "Definitions:"
+  echo "  <conf-dir>                  Override CELEBORN_CONF_DIR"
+  echo "  <celeborn-class-name>       The class name of the celeborn command to run"
```
Comment (pan3793) [inline sbin/celeborn-daemon.sh:35]:
```suggestion
  echo "  <celeborn-class-name>       The main class name of the celeborn command to run"
```

File: sbin/celeborn-daemon.sh [hunk 2]
```diff
@@ -28,20 +28,22 @@
 #   CELEBORN_NO_DAEMONIZE   If set, will run the proposed command in the foreground. It will not output a PID file.
 ##
 
-usage="Usage: celeborn-daemon.sh [--config <conf-dir>] (start|stop|status) <celeborn-command> <celeborn-instance-number> <args...>"
+usage() {
+  echo "Usage: celeborn-daemon.sh [--config <conf-dir>] (start|stop|restart|status) <celeborn-class-name> <celeborn-instance-number> [args]"
+  echo "Definitions:"
```
Comment (pan3793) [inline sbin/celeborn-daemon.sh:33]:
```suggestion
  echo ""
  echo "OPTIONS"
```

File: sbin/celeborn-daemon.sh [hunk 3]
```diff
@@ -28,20 +28,23 @@
 #   CELEBORN_NO_DAEMONIZE   If set, will run the proposed command in the foreground. It will not output a PID file.
 ##
 
-usage="Usage: celeborn-daemon.sh [--config <conf-dir>] (start|stop|status) <celeborn-command> <celeborn-instance-number> <args...>"
+usage() {
+  echo "Usage: celeborn-daemon.sh [--config <conf-dir>] (start|stop|restart|status) <celeborn-class-name> <celeborn-instance-number> [args]"
+  echo ""
+  echo "OPTIONS"
```
Comment (pan3793) [inline sbin/celeborn-daemon.sh:34]:
```suggestion
  echo "Options:"
```

**Comment by waitinfuture** [comment]
Thanks, merging to main

**Pull request closed without merging**